serde-generate/src/ocaml.rs

```diff
@@ -0,0 +1,280 @@
+use crate::{
+    indent::{IndentConfig, IndentedWriter},
+    CodeGeneratorConfig,
+};
+use heck::SnakeCase;
+use include_dir::include_dir as include_directory;
+use serde_reflection::{ContainerFormat, Format, Named, Registry, VariantFormat};
+use std::{
+    collections::{BTreeMap},
+    io::{Result, Write},
+    path::PathBuf,
+};
+
+fn capitalize(s: &str) -> String {
```

Comment:
I was using for 2 cases:
- for external libraries: in this case we need foo_lib -> Foo_lib (in the simple case. I should probably give some better options of configuration since the lib and modules are sometime not the same)
- for variant names: in this case I can use pascal case (which is called camel case in heck.0.3.2, it would probably be a good idea to upgrade to 0.4.0 to have the right names)

Reply:
Makes sense. Perhaps just move this function to `common.rs` then?